src/components/Overlay/README.md

```diff
@@ -0,0 +1,32 @@
+<!--GITHUB_BLOCK-->
+
+# Loader
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Overlay} from '@gravity-ui/uikit';
+```
+
+The `Overlay` component renders an overlay over the parent element with relative position.
+For example, it can be used to preserve the desired layout while loading data.
+
+### PropTypes
+
+| Name      | Type               | Required | Default | Description                         |
```

Comment:
Please follow the structure we use in other readme files, see `Button`'s readme as an example